### PR TITLE
announce 4.69

### PR DESCRIPTION
**Pull request by fredemmott**

### CODE REVIEW

File: _posts/2020-08-04-hhvm-4.69.markdown [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+HHVM 4.63&ndash;4.68 remain supported, as do the 4.32 and 4.56 LTS releases.
+
+# Breaking Changes
+
+- Directly implementing the same interface twice with multiple generics is now
+  banned, due to incorrect over-permissiveness.
+- The `hhvm.check_return_type_hints` runtime option has been removed.
+- It is now a typechecker error to attempt to directly construct an
+  `IntlIterator`.
+- It is now a type error to have duplicate classes, type aliases, or records
```
Comment (lexidor) [inline _posts/2020-08-04-hhvm-4.69.markdown:18]: I don't quite understand. It was already an error to have duplicate classes or type aliasses.
What changes here?

File: _posts/2020-08-04-hhvm-4.69.markdown [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+title: "HHVM 4.69"
+layout: post
+author: fred
+category: blog
+---
+
+HHVM 4.69 is released! This release marks the end of support for 4.62;
+HHVM 4.63&ndash;4.68 remain supported, as do the 4.32 and 4.56 LTS releases.
```
Comment (jjergus) [inline _posts/2020-08-04-hhvm-4.69.markdown:9]: 4.63 was skipped

File: _posts/2020-08-04-hhvm-4.69.markdown [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+# Breaking Changes
+
+- Directly implementing the same interface twice with multiple generics is now
+  banned, due to incorrect over-permissiveness.
```
Comment (jjergus) [inline _posts/2020-08-04-hhvm-4.69.markdown:14]: I don't quite understand this one, an example could be useful.

File: _posts/2020-08-04-hhvm-4.69.markdown (hunk 1 above)
Comment (jjergus) [reply _posts/2020-08-04-hhvm-4.69.markdown:18]: +1 an example or clarification would be useful

File: _posts/2020-08-04-hhvm-4.69.markdown (hunk 3 above)
Comment (fredemmott) [reply _posts/2020-08-04-hhvm-4.69.markdown:14]: will update - `class foo implements I<string>, I<int> {}` would only properly enforce `I<string>`

File: _posts/2020-08-04-hhvm-4.69.markdown (hunk 1 above)
Comment (fredemmott) [reply _posts/2020-08-04-hhvm-4.69.markdown:18]: Repo-auth only (dfe31524392e29a9bde87188fe3b38850104b32e), updating

Comment (jjergus) [reply _posts/2020-08-04-hhvm-4.69.markdown]: so a compilation error, not a typechecker error?

Comment (fredemmott) [reply _posts/2020-08-04-hhvm-4.69.markdown]: yep

Comment (Wilfred) [reply _posts/2020-08-04-hhvm-4.69.markdown]: https://github.com/facebook/hhvm/commit/dfe31524392e29a9bde87188fe3b38850104b32e

**Pull request merged**